### ISSUE TITLE
Add option for reversible legend

### ISSUE THREAD

**Issue by matschaffer** [panodata/grafana-map-panel#85]
We have a panel up at https://grafana.safecast.cc/d/7wsttvxGk/safecast-airnote?orgId=1&var-device_urn=note:dev:864475041076562 and got feedback the legend seems backwards (low at the top, high at the bottom).

Looks like we can probably add `.reverse()` to https://github.com/panodata/grafana-map-panel/blob/5e7ae9fbebf0bd1cebd18e5c0a79c4e3217a1f42/src/worldmap.ts#L129 but might be better to make it an option if people prefer the current order.

**Comment by amotl** [comment]
Dear Mat,

your suggestion looks very reasonable to me. We will be happy to accept further contributions around these details.

With kind regards,
Andreas.

**Comment by nokton** [comment]
Yup, @nokton preferred it that way so it’s in line with our radiation maps. Not sure if you want to keep a .reverse but may come in handy in the future. Prefer the new ordering as the default!
